src/db/Kpi/Goal/Goal.js

```diff
@@ -0,0 +1,41 @@
+import { db } from '@/config/firebaseConfig';
+import props from './props';
+import { validateCreateProps, validateUpdateProps } from '../../common';
+
+async function create(kpiId, data) {
+  if (!validateCreateProps(props, data)) {
+    throw new Error('Invalid data');
+  }
+
+  const { ref } = await db.collection('kpis').doc(kpiId).get();
+  const goalsRef = ref.collection('goals');
+
+  return goalsRef.add(data);
```

Comment:
Bør ikke bruke `createDocument` (`src/db/common/methods/updateDocument.js`, og tilsvarende `updateDocument`)  for å få med bl.a. `created/createdBy` metadata på objektet? Eller anser vi det som irrelevant her? (Antar det ikke er aktuelt i noen visning eller noe nå, men kanskje for litt delvis "audit"?)